source/_integrations/xiaomi_ble.markdown

```diff
@@ -49,3 +49,21 @@ There are a few ways to obtain a bindkey for your device:
 
 * Set your own. The [Telink Flasher](https://atc1441.github.io/TelinkFlasher.html) allows you to generate new bindkeys for devices it supports. The new bind key will work with Home Assistant, but the Mi Home app will not recognize the sensor anymore once the device has been activated by the TeLink flasher application. To use the sensor again with the Xiaomi Mi Home app, the device needs to be removed and then re-added inside the Mi Home app.
 * Extract the keys from Xiaomi Cloud using a [token extractor](https://github.com/PiotrMachowski/Xiaomi-cloud-tokens-extractor) tool.
+
+## Devices
+
+### HHCCJCY01 plant sensor
+
+HHCCJCY01, also known as MiFlora or "Flower Care", should be automatically discovered. However, if the firmware is too old, it won't send the right BLE beacons and an update via the app is required.
+
+Flower Care firmware update steps:
+
+* Install the official "Flower Care" app by HHCC:
+  * [Google Play](https://play.google.com/store/apps/details?id=com.huahuacaocao.flowercare) (requires location and storage permission)
+  * [Apple App Store](https://apps.apple.com/us/app/flower-care/id1095274672)
+* Place the device with the app within 10cm of the plant sensor
+* Press the "+" button on the top right in the devices tab of the app
+* Add the plant sensor to the app and select an arbitrary plant
+* Wait for the synchronisation of the sensor to finish and a dialog asking for a firmware update should appear (this might take a few minutes)
```

Comment:
```suggestion
* Wait for the synchronization of the sensor to finish, and a dialog asking for a firmware update should appear (this might take a few minutes)
```